### DISCUSSION TITLE
ReferenceError: TextEncoder is not defined

### DISCUSSION THREAD

**Discussion by digvijay86** [mswjs/msw#2234] [Q&A] [answered]
How to solve the error ReferenceError: TextEncoder is not defined?

**Comment by kettanaito** [comment, marked as answer]
Hi. Here you go: https://mswjs.io/docs/migrations/1.x-to-2.x#requestresponsetextencoder-is-not-defined-jest

You can also use https://github.com/mswjs/jest-fixed-jsdom instead.
